### DISCUSSION TITLE
Display entry author list as searchable dropdown, like relationships, instead of simple list of radio buttons

### DISCUSSION THREAD

**Discussion by shbchk** [ExpressionEngine/ExpressionEngine#3796] [Feature Requests]
### Suggest an idea for ExpressionEngine

I expect this list to be quite extensive, it feels like a list of radio buttons is not the best choice for it. 
![image](https://github.com/ExpressionEngine/ExpressionEngine/assets/12234507/579bc66c-97e1-413c-b7bd-24447a55879d)

**Comment by intoeetive** [comment]
This would turn into searchable select once you have more that certain number of authors (I think 8)
